### DISCUSSION TITLE
Have I been IP-blocked from Vercel-hosted sites?

### DISCUSSION THREAD

**Discussion by thcrt** [vercel/community#4764] [Account Issue] [answered]
### Question

Roughly a week ago, I found that I was unable to connect to a website I visited on a semi-regular basis ([NixOS and Flakes](https://nixos-and-flakes.thiscute.world/), a guide to NixOS). I chalked it down to a misconfiguration on the part of the website's owner. But a few days ago, I noticed some other websites wouldn't load for me either. Out of curiosity, I ran `dig` on the domain names (a program that gives the IP address associated to a domain) and it returned `76.76.21.21` -- an IP used by Vercel. All of the websites I'm unable to access are hosted at this IP by Vercel. This includes python-poetry.org which I use in development.

I believe Vercel is blocking connections from my home IP address. Using a VPN, I have no issue connecting to any of these sites. Normally, though, I see "connection refused". This persists no matter what configuration or what device I'm on. 

I don't know why my IP has been blocked. I've never used Vercel hosting or done anything unusual on a Vercel-hosted website. For some context, I live in student accomodation in the Netherlands shared between a few hundred students, in case unusually high traffic is coming from my IP, but I'm not aware of any reason for a block.

I've emailed Vercel support but received no answer. Is there anyone here who can create a ticket for me or give any further support?

Thanks :)

**Comment by thcrt** [comment]
@amyegan this is quite frustrating since a few tools I rely on are hosted on Vercel. I haven't yet received any communication from anyone on the team. Please could I get some information on what's going on?

Thanks :)

**Reply by amyegan** [reply]
Sorry you're running into trouble with site access. I don't know of any general block from Vercel, but an individual site may have used [Deployment Protection](https://vercel.com/docs/security/deployment-protection) to limit access. Or sometimes a network admin or ISP might block us due to the wide range of things that folks deploy on Vercel. 

If you let me know your case number, I can check on the ticket for you 🙏

**Reply by thcrt** [reply]
Thanks for the response :) I don't have a case number -- I emailed `registration@vercel.com` (none of the contact emails listed quite fit my problem so I chose registration) and received no response.

**Reply by thcrt** [reply]
I can't access:
- `phuoc.ng`
- `python-poetry.org`
- `thiscute.world`
I'm sure this isn't an exhaustive list, I know for a fact there are others that I can't access.

The Vercel main page loads fine.

**Reply by thcrt** [reply]
@amyegan any updates? :)

**Reply by thcrt** [reply]
@amyegan `getfleek.dev` is similarly inaccessible for me. I'd really, really appreciate some support with this.

**Reply by amyegan** [reply]
Without a the case number you got from that email I'm not able to identify you in the system to investigate. Since you reported that the sites are accessible again, I suspect something else was blocking you from accessing Vercel hosted sites. I'm happy to hear it is no longer a problem for you! 🙂

**Reply by thcrt** [reply]
@amyegan Like I said, unfortunately I received no response from any email, so I didn't get a case number. 🤷🏻

**Comment by thcrt** [comment, marked as answer]
@amyegan Issue seems to be resolved now. I still haven't heard anything from Vercel. Obviously, I hope this doesn't happen again, but if it does I'd appreciate more transparency in the process -- being arbitrarily blocked and unblocked from a good portion of the web isn't a super fun experience, haha.

All the best!

**Comment by swish933** [comment]
It seems like this is a recurring issue. I have a site deployed on Vercel. It worked fine up until recently. I can only open the site when i use a vpn.

**Reply by darphiz** [reply]
Same here I can't access my website hosted on vercel without a VPN

**Reply by amyegan** [reply]
There have been some recent reports of ISP level blocks in Nigeria which would cause this kind of problem. Are you unable to access all Vercel sites, or is this only happening for a specific one?

**Reply by swish933** [reply]
I'm able to access some Vercel sites but i also can't access some

**Reply by amyegan** [reply]
Would you be able to share some diagnostic information to help us identify where the block is happening? The following test commands will provide ISP and route information so we can raise the issue with the relevant upstream providers.

Linux/Mac terminal commands: 
```
# run these commands from the same (problematic) network/machine

# This shows the reporter's global IP address
$ curl https://ipinfo.io/

# Test reachability to 76.76.21.21
$ ping -c 4 76.76.21.21

# Test reachability to cname.vercel-dns.com
$ ping -c 4 cname.vercel-dns.com

# Dig your domain
# Do not include "https://" (e.g., just type "dig example.com")
$ dig ${replace-with-your-domain}

# Dig your domain via public DNS
# Do not include "https://" (e.g., just type "dig example.com @8.8.8.8")
$ dig ${replace-with-your-domain} @8.8.8.8

# curl content
$ curl -sv https://${replace-with-your-domain}

# traceroute or mtr result
$ sudo mtr -wr -c 20 76.76.21.21
```


Windows Powershell commands:
```
# This shows the reporter's global IP address
> curl.exe https://ipinfo.io/

# Test reachability to 76.76.21.21
> ping -n 4 76.76.21.21

# Test reachability to cname.vercel-dns.com
> ping -n 4 cname.vercel-dns.com

# DNS resolve your domain
# Do not include "https://" (e.g., just type "nslookup example.com")
> nslookup ${replace-with-your-domain}

# DNS resolve you domain via public DNS
# Do not include "https://" (e.g., just type "nslookup example.com 8.8.8.8")
> nslookup ${replace-with-your-domain}

# curl content
> curl.exe -sv https://${replace-with-your-domain}

# traceroute
> tracert 76.76.21.21
```

**Reply by swish933** [reply]
Alright do i share that info here

**Reply by swish933** [reply]
"ip": "102.88.82.240",
  "city": "Lagos",
  "region": "Lagos",
  "country": "NG",
  "loc": "",
  "org": "AS29465 MTN NIGERIA Communication limited",
  "timezone": "Africa/Lagos",
  "readme": "https://ipinfo.io/missingauth"

---

PING 76.76.21.21 (76.76.21.21): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2

--- 76.76.21.21 ping statistics ---
4 packets transmitted, 0 packets received, 100.0% packet loss

---

PING cname.vercel-dns.com (76.76.21.22): 56 data bytes
64 bytes from 76.76.21.22: icmp_seq=0 ttl=244 time=75.370 ms
64 bytes from 76.76.21.22: icmp_seq=1 ttl=244 time=75.009 ms
64 bytes from 76.76.21.22: icmp_seq=2 ttl=244 time=129.844 ms
64 bytes from 76.76.21.22: icmp_seq=3 ttl=244 time=78.723 ms

--- cname.vercel-dns.com ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 75.009/89.737/129.844/23.201 ms

---

; <<>> DiG 9.10.6 <<>> georgeikem.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 17160
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;georgeikem.com.			IN	A

;; ANSWER SECTION:
georgeikem.com.		1684	IN	A	76.76.21.21

;; Query time: 25 msec
;; SERVER: 172.20.10.1#53(172.20.10.1)
;; WHEN: Wed Mar 13 20:28:48 WAT 2024
;; MSG SIZE  rcvd: 59

---

; <<>> DiG 9.10.6 <<>> georgeikem.com@8.8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 9975
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;georgeikem.com\@8.8.8.8.8.	IN	A

;; AUTHORITY SECTION:
.			3600	IN	SOA	a.root-servers.net. nstld.verisign-grs.com. 2024031300 1800 900 604800 86400

;; Query time: 77 msec
;; SERVER: 172.20.10.1#53(172.20.10.1)
;; WHEN: Wed Mar 13 20:29:55 WAT 2024
;; MSG SIZE  rcvd: 128

---

*   Trying 76.76.21.21...
* TCP_NODELAY set
* Connection failed
* connect to 76.76.21.21 port 443 failed: Operation timed out
* Failed to connect to georgeikem.com port 443: Operation timed out
* Closing connection 0

---



Start: 2024-03-13T21:59:56+0100
|HOST: My-Mac.local  | Loss%  | Snt  | Last   |Avg    | Best    | Wrst   | StDev|
|---------------------|--------|-----|-------|------|-------|-------|-------|
| 1. 172.20.10.1                  | 0.0%     | 20   |  5.1    | 2.8    | 1.9       | 5.1      | 0.8     |
|  2. ???                             | 100.0    |  20   | 0.0    | 0.0    | 0.0      | 0.0      | 0.0     |
|  3. 10.170.129.173          | 0.0%     | 20    | 56.0  | 61.0  | 35.0    | 217.3  | 42.9   |
|  4. ???                             |100.0     | 20    | 0.0    | 0.0    | 0.0      | 0.0      | 0.0     |

**Reply by amyegan** [reply]
Thank you 🙇 

I shared this info with the engineers working on the issue.

**Reply by smaeda-ks** [reply]
Do you all still see the problem? Just trying to see the latest here

**Reply by okore-lens** [reply]
I am also unable to access all vercel hosted sites using one of my ISP providers.  Is there any known solution to this? It started about a week ago
![image](https://github.com/vercel/community/assets/91420257/dcabbe40-ec38-4473-8791-b2e0d1953ef0)

**Reply by swish933** [reply]
@okore-lens I had to move my project off vercel. So I'm not sure if there's a solution yet

**Reply by okore-lens** [reply]
Mine was resolved yesterday

**Reply by swish933** [reply]
What happened? And how was it resolved?

**Reply by vonba** [reply]
I see that this issue is quite old, but I have the same problem. My results are the same as swish933 above: 100% packet loss and NOERROR status on the domain query. I can't access my deployment on Vercel (or any of the websites mentioned in this thread) since more than a week, and interestingly I can't even access vercel.com. I'm located in Mexico City and the above only happens when I'm on my home wifi.

@amyegan was there ever a solution to this issue? Is Vercel just blocking people around the world willy nilly?

**Comment by priyank56** [comment]
Hello, I just got into a similar experience.

I am unable to access any Vercel domain, sub domain, or sites deployed in vercel.

Anyone knows how to resolve it?

My all projects are been deployed here - and I am unable to access those.

I have checked with different device, it seems like - my wifi ip has been blocked. I can not access any of my site or other sites deployed in vercel. When I switched on to my personal internet connection, I can get an access.

Any solution?

**Reply by priyank56** [reply]
I am trying to access below site:

My HTML online Gaming site.
[102.play.online.zupeegame.in](https://102.play.online.zupeegame.in)
[mobile.zupeegame.in](https://mobile.zupeegame.in)

And another domain:
For creating resume online for free.
[snowresume.com](https://snowresume.com)

I can not recognise any suspicious activity done from my side.

**Comment by darphiz** [comment]
When I had the issue, the solution was to switch to another provider

[a comment by an automated account is not shown]
